bolt-service/test/request.json

```diff
@@ -0,0 +1,23 @@
+{
+  "target": {
+    "hostname": "test.delivery.puppetlabs.net",
+    "user": "root",
+    "password": "Q*****!",
+    "options": {
```

Comment:
We've also updated the API: https://github.com/puppetlabs/bolt/blob/master/developer-docs/bolt_server.md#post-sshrun_task - no more 'options' key